client/src/LayoutComponents/CohortDetails/CohortDetails.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from "react";
+import { Slide } from "react-slideshow-image";
+import ProjectView from '../../SharedComponents/ProjectView'
+import StudentCard from '../../SharedComponents/StudentCard'
+import "./CohortDetails.css";
+import axios from "axios";
+
+
+class CohortDetails extends Component {
+  state = {
+    cohorts: [],
+  };
+  componentDidMount = () => {
+    axios
+      .get("/api/cohortinfo")
+      .then((result) => {
+        console.log(result.data);
+        const resultarray = result.data.map((e) => e);
```

Comment:
@Hussein-shahatet97 I think the map() isn't actually doing anything, so you could remove it:
`const resultarray = result.data;`

Reply:
thank you for your review,i will make changes by removing map from function and rewrite it into render.